app.go

```
@@ -139,10 +139,14 @@ func (a *App) Run() error {
 	if err = eg.Wait(); err != nil && !errors.Is(err, context.Canceled) {
 		return err
 	}
-	for _, fn := range a.opts.afterStop {
-		err = fn(sctx)
+
+	if len(a.opts.afterStop) != 0 {
+		for _, fn := range a.opts.afterStop {
+			err = fn(sctx)
```

ignore err and continue the loop? or should we use log output the err?